src/main/java/mops/presentation/ExceptionController.java

```diff
@@ -52,7 +52,11 @@ public ModelAndView resolveException(HttpServletRequest request,
     @RequestMapping("/material1/error")
     @SuppressWarnings({ "PMD.LawOfDemeter", "PMD.DataflowAnomalyAnalysis", "PMD.CyclomaticComplexity" })
     public String handleError(KeycloakAuthenticationToken token, HttpServletRequest request, Model model) {
-        Account account = Account.of(token);
+        if (token != null) {
```

Comment:
The token should not be empty?

Reply:
i had errors in my console, couldn't reproduce later tho